hs_core/search_indexes.py

```diff
@@ -0,0 +1,86 @@
+from haystack import indexes
+from hs_core.models import BaseResource
+
+
+class BaseResourceIndex(indexes.SearchIndex, indexes.Indexable):
+    text = indexes.CharField(document=True, use_template=True)
+    short_id = indexes.CharField(model_attr='short_id')
+    doi = indexes.CharField(model_attr='doi', null=True)
+    author = indexes.CharField(model_attr='first_creator', faceted=True)
+    title = indexes.CharField(model_attr='title', faceted=True)
+    abstract = indexes.CharField(model_attr='description')
+    creators = indexes.MultiValueField(faceted=True)
+    contributors = indexes.MultiValueField()
+    subjects = indexes.MultiValueField(faceted=True)
+    public = indexes.BooleanField(model_attr='raccess__public', faceted=True)
+    discoverable = indexes.BooleanField(faceted=True)
+    created = indexes.DateTimeField(model_attr='created')
+    modified = indexes.DateTimeField(model_attr='updated')
+    organization = indexes.CharField(faceted=True, null=True)
+    author_email = indexes.CharField()
+    publisher = indexes.CharField(faceted=True)
+    rating = indexes.IntegerField(model_attr='rating_sum')
+    coverages = indexes.MultiValueField()
+    formats = indexes.MultiValueField()
+    identifiers = indexes.MultiValueField()
+    language = indexes.CharField(faceted=True)
+    sources = indexes.MultiValueField()
+    relations = indexes.MultiValueField()
+    #type = indexes.CharField()
```

Comment:
The 'type' attribute cannot be indexes. It gives the error 'Error: ‘'Type' object has no attribute 'value'”'. I think there might be some problem in the 'class Type(AbstractMetaDataElement):' in the model.py file. We mentioned about it in our last Wed's meeting.